### DISCUSSION TITLE
Adguard Home - Question concerning LXC upgrades, from tteck to community-scripts

### DISCUSSION THREAD

**Discussion by pbn42** [community-scripts/ProxmoxVE#396] [Q&A] [answered]
Hi,

I configured a lots of things on proxmox, (huge) thanks to tteck scripts.

When i wanted to upgrade my AdguardHome LXC, i usualy just re-run the tteck commande in my terminal :

`bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/adguard.sh)"`
=>
![image](https://github.com/user-attachments/assets/871d6d0d-952f-495e-8f69-cda7ccd4579a)

I tried running the script from community-scripts instead :

`bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/ct/adguard.sh)"`
=> 
![image](https://github.com/user-attachments/assets/3f68790e-094d-4a6c-8c79-b90db4dbf46b)

Can somebody please tell me what to do please ? I saw the message on the community page "Adguard Home can be updated via the user interface.", bud couldn't find where to do it.

Thanks a lot !

**Comment by MickLesk** [comment, marked as answer]
The reason was quite simple, AdguardHome offers a practical update function, so you don't need a high-maintenance script. Especially since it also works faster and better.

Open your Adguard-Home Installation in Browser, for example: 192.168.2.100
Then Login.

If you have an update, you see it on the top of the page, otherwise you can switch to bottom and search.
![image](https://github.com/user-attachments/assets/212b6605-c6ea-4148-8eee-d310b7169655)

**Reply by pbn42** [reply]
Thanks a lot for your explanations !

**Reply by shyndman** [reply]
Well...hold on now. Is that really practical?

**Reply by MickLesk** [reply]
i use this since 3 years without any issue

**Comment by shyndman** [comment]
I don't doubt it. Because automation...who needs it, right?
